### DISCUSSION TITLE
An Update, but not an Announcement

### DISCUSSION THREAD

**Discussion by jon-harper** [jon-harper/OmniBox#84] [Ideas]
Happy holidays! \*

To start, I want to welcome...ah, basically everyone! I haven't posted anything here since 0.9.9 release, so this is my first chance to say hi to newcomers.

If you're building an OmniBox and have questions that you can't find in the documentation, please ask! Knowing what issues folks run into helps me improve the next release and the documentation. I'm always around for support if you need it.

On a quick personal note, I took some time off after release to care for my mom. She had a very successful knee replacement, but you can't do much for the first few weeks after surgery. She's regained most of her mobility, so my eyes are wandering back to OmniBox.

-----

I've tagged this post as a question and "good first issue" because I'd like to hear from everyone.

## Newly Built OmniBoxes

If you've built an OmniBox, please share!

- How did it go overall?
- What was your most significant issue (or issues) in assembly?
- Did you find the documentation easy to navigate and helpful?
- Do you have any suggested improvements?
- Lastly, share pics if you'd like! I'd love to see everyone's configurations!

## Social Media

I shut down the OmniBox Twitter page. I don't want to delve into the tension around Twitter in the news, nor do I want to create controversy about where I make announcements regarding OmniBox.

In the short term, posts will be here on the GitHub Discussion page or on Reddit. I'm also experimenting with [Post.news](https://post.news/jonspaceharper) and [Mastodon](https://mastodon.online/@jonspaceharper), but neither are satisfactory right now.

I *had* an Instagram page. If there is interest, I would consider re-opening it. A Facebook group is also an option. Suggestions are welcome if you have an idea or preference.

**EDIT** I created a Discord, since this was requested before and is popular. [Join link here](https://discord.gg/nKmJZMhfz5).

## The Future

I have ideas. I'm running a test print for one right now, but announcements will come in the new year.

\* (If you celebrate a holiday this time of year. Otherwise, happy Saturday!) 🙂

**Comment by phillyaws** [comment]
I've finally almost completely finished mine!  Just need to print out a few changes I made, and I'll share pictures!

**Reply by jon-harper** [reply]
Sweet! Looking forward to it!

**Reply by jon-harper** [reply]
My brain connected the dots! Your stuff on Printables is great.

This may interest you since you're doing panel mounts: I standardized the wiring orders for my panel mounts for interchangeability. I've documented a lot of it and related stuff; the [pages about panel mount and connectors might be useful to you](https://jon-harper.github.io/clockmaker-project/wiring/panel_tips/).

Hope that helps and happy new year!

**Comment by Otann** [comment]
I've finally completed my build. It took me a while to dial in my printer to work well with a somewhat demanding brand of PLA, but all worked well in the end!

Since I had an Ender 3 v2 and the supplied screen didn't work with klipper, I've adapted the Raspberry Touch Screen that I had spare. And designed a plate to work with it. I highly recommend any sort of touchscreen over the rotating knob.

I like it a lot, thank you again for the brilliant project! 🙏 

The documentation is easy to prepare and to follow. With all parts printed and crimping tools prepared, it took me about a couple of hours to complete. And couple of days to adapt a screen.

Now onto an enclosure ✊ 
Do you maybe have any tips or links to other projects? (besides LACK enclosures)

And here are some pictures:

![From the distance](https://user-images.githubusercontent.com/683786/212431206-8d8ba154-c937-475f-9563-feecd82570ff.jpeg)

![From the right](https://user-images.githubusercontent.com/683786/212431220-ef435a6c-7e70-4f47-89eb-5690464043ec.jpeg)

![From the left](https://user-images.githubusercontent.com/683786/212431231-4912ed69-d29b-4e6a-8acd-1cf5ea5f5d6a.jpeg)

![Profile close up](https://user-images.githubusercontent.com/683786/212431243-4c16df44-3e9a-463b-a954-7fd5496ce304.jpeg)

**Comment by jon-harper** [comment]
That looks fantastic! Thanks for sharing. 😊

 

I don’t have specific enclosure suggestions, but if you’re building your own I’ve at least begun compiling data as part of this project <https://jon-harper.github.io/clockmaker-project/> . There’s not much there outside of electrical section, however—it’s fairly early days.

**Reply by Otann** [reply]
Nice! I admire your ability to document things as you go :)

I can't buy from RatRig unfortunately, but I plan to design something similar and see how much it would cost me. If I'd have time, I'd contribute there to according to my abilities ;)
